src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -855,10 +870,25 @@ public void addCollaborators(GHUser... users) throws IOException {
      * @throws IOException
      *             the io exception
      */
-    public void addCollaborators(Collection<GHUser> users) throws IOException {
+    @Deprecated
+    public void addCollaborators(List<GHUser> users) throws IOException {
```

Comment:
This would be an API change.
```suggestion
    public void addCollaborators(Collection<GHUser> users) throws IOException {
```